tests/playwright/sanity/basic-gallery.test.ts

```diff
@@ -60,6 +62,34 @@ test( 'Basic Gallery Lightbox test with older Swiper', async ( { page, apiReques
 	await testBasicSwiperGallery( editor );
 } );
 
+test.only( 'Basic Gallery Lightbox test with breakpoints', async ( { page, apiRequests }, testInfo ) => {
+	// Arrange.
+	const wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
+	const editor = await wpAdmin.openNewPage();
+	const breakpoints = new Breakpoints( page );
+
+	await editor.closeNavigatorIfOpen();
+
+	await editor.addWidget( 'image-gallery' );
+	await editor.openPanelTab( 'content' );
+	await editor.addImagesToGalleryControl();
+	await editor.setSelectControlValue( 'open_lightbox', 'yes' );
+
+	await editor.publishPage();
+
+	await breakpoints.setBreakpoint( editor, 'mobile', viewportSize.mobile.width );
+
+	// Act.
+	const pageId = await editor.getPageId();
+	await page.goto( `/?p=${ pageId }` );
+	await page.setViewportSize( viewportSize.mobile );
+	await page.locator( 'div#gallery-1 img' ).first().click();
+	await editor.page.waitForTimeout( 1000 );
+
+	// Assert.
```

Comment:
For clarity: is the padding of `70px 0` applied within this screenshot? Would it fail, if the breakpoint wouldn't work?